client/state/action-types.js

```diff
@@ -29,7 +29,11 @@ export const ANALYTICS_PAGE_VIEW_RECORD = 'ANALYTICS_PAGE_VIEW_RECORD';
 export const ANALYTICS_STAT_BUMP = 'ANALYTICS_STAT_BUMP';
 export const AUTOMATED_TRANSFER_ELIGIBILITY_REQUEST = 'AUTOMATED_TRANSFER_ELIGIBILITY_REQUEST';
 export const AUTOMATED_TRANSFER_ELIGIBILITY_UPDATE = 'AUTOMATED_TRANSFER_ELIGIBILITY_UPDATE';
+export const AUTOMATED_TRANSFER_INITIATE = 'AUTOMATED_TRANSFER_INITIATE';
```

Comment:
Thanks @vindl. Do you know specifically who is working on that? The endpoint itself handles both cases of plugins and themes so I think it's probably appropriate here too. Either way, the intention is _initiate a transfer_ and not _initiate a plugin transfer_ or _initiate a theme transfer_ but I could envision separate action creators.